test/performance/profiles.jmx

```diff
@@ -101,7 +101,53 @@
           <stringProp name="HTTPSampler.response_timeout"></stringProp>
           <stringProp name="TestPlan.comments">Detected the start of a redirect chain</stringProp>
         </HTTPSamplerProxy>
-        <hashTree/>
+        <hashTree>
+          <ResponseAssertion guiclass="AssertionGui" testclass="ResponseAssertion" testname="Response Assertion" enabled="true">
+            <collectionProp name="Asserion.test_strings">
+              <stringProp name="1235244996">Page not found</stringProp>
```

Comment:
@markysoft I've done some more testing and I don't think there needs to be an explicit check for the 404 as this is done by default. I thought the exit code was being set to non-zero when there is an assertion check but this isn't the case so it makes no difference. Hopefully TeamCity can pick up on the number of tests that have failed from the report.